docs/source/dev_resources/tutorials/new_vipm_package/index.rst

```diff
@@ -0,0 +1,144 @@
+   Launch the VI Package Manager application.
+
+2. **Create a New Package**  
+
+   - Click the **Open VI Package Builder** icon in the VIPM toolbar (top-left).
+   - In the Package Builder menu, select **Create New**.
+   - Set the **Source Directory** to the root folder of your LabVIEW project.
+   - Set the **Product Name** to a meaningful name (e.g., `"IR Sensor Plugin"`).
+   - Set the **Company Name** to:
+
+    - Rxn Rover if you're a first-party developer.
+    - Your own company name if you're a third-party developer.
+
+   This will create a new ``.vipb`` (VI Package Builder) file named after the Product Name in your project directory.
+
+3. **Configure Basic Build Information**  
+
+   - **Source Directory**: Points to your LabVIEW project root.
+   - **Build Output Directory**: Choose a location like ``<your-workspace>\VIPMPackages``.
+
+.. note::
+   The rest of the configuration differs slightly depending on whether you're a **first-party** (Rxn Rover) or **third-party** developer.
+
+First-Party Plugins (Rxn Rover Developers)
+------------------------------------------
+
+These instructions are for Rxn Rover developers creating official plugins or drivers.
+
+1. **Display Information**
+
+  - **Product Name**: Name of your plugin (e.g., "IR Sensor Plugin").
+  - **Icon**: Optional.
+  - **Company Name**: Rxn Rover
+  - **Author Name**: Rxn Rover
+  - **Product Homepage**: ``https://github.com/RxnRover/<your-plugin-repo>``
+  - **Copyright**: Copyright (c) `<year>` RxnRover
+  - **License Agreement**: Path to ``LICENSE.txt`` in your project folder.
+  - **Summary**: Short description (e.g., "A plugin for IR sensor data acquisition and processing").
+
+2. **Destinations**
+
+  - Set install location to:
+    - ``Documents\RxnRover\Plugins\<plugin-type>\<Product Name>`` for plugins.
+    - ``instr.lib\RxnRover\<Product Name>`` for drivers.
+
+  - **Package Filename**:  
+    Use this format:  
+    ``rxn_rover_lib_<product_name>-X.X.X.X.vip``
+
+Continue with the rest of the tutorial at :ref:`build-and-share-package`.
+
+Third-Party Plugins
+-------------------
+
+For external developers who wish to make their plugin compatible with Rxn Rover.
+
+1. **Display Information**
+
+  - **Product Name**: Name of your plugin (e.g., "IR Sensor Plugin").
+  - **Icon**: Optional.
+  - **Company Name**: `Your organization's name`
+  - **Author Name**: `Your personal or your organization's name`
+  - **Product Homepage**: e.g., ``https://github.com/<github_username>/<your_plugin_repo.>``
+  - **Copyright**: Copyright (c) `<year>` `<Your Company>`
+  - **License Agreement**: Path to ``LICENSE.txt`` in your project folder.
+  - **Summary**: A brief description of the plugin.
+  - **Description**: Provide a more detailed explanation of the plugin’s features and how to use it.
+
+2. **Destinations**
+
+  - For plugins:  
+    ``Documents\RxnRover\Plugins\<plugin-type>\<Product Name>``
+
+  - For drivers:  
+    ``instr.lib\RxnRover\<Product Name>``
```

Comment:
Most people actually seem to put their drivers straight into `instr.lib\<Product Name>`. The `instr.lib\RxnRover` subfolder is actually just a convention I use to make ours easy to find. This should be changed to either `instr.lib\<Product Name>`, `instr.lib\<Company Name>\<Product Name>`, or both.
```suggestion
  - For drivers:  
    ``instr.lib\RxnRover\<Product Name>``
```